velox/exec/Driver.cpp

```diff
@@ -615,6 +615,11 @@ void Driver::closeOperators() {
   // Add operator stats to the task.
   for (auto& op : operators_) {
     auto stats = op->stats(true);
+    auto backgroundCpuTimeMs = op->getBackgroundCpuTimeMs();
```

Comment:
Can we add this handling in Operator::Close instead of here? We can just make sure all the derived operator close method call the base class close method? Thanks!

Reply:
Fixed